.rubocop.yml

```diff
@@ -27,6 +27,7 @@ Metrics/BlockLength:
 
 Metrics/CyclomaticComplexity:
   Exclude:
+    - 'app/models/user.rb'
```

Comment:
With the change you in `user.rb`, pretty sure this can go away now.

Reply:
Yeah, i'm still whack-a-moling this thing :(

Reply:
Turns out, user.rb is still complex, it's only over by one, i'm tempted to leave it?

Reply:
Ah — I guess that single `||` in the ORCID assignment pushed Rubocop over the edge 🤯 

I'm fine with leaving this.